src/tools/pub/cmd/pub-outdated.md

```diff
@@ -1,65 +1,69 @@
 
 {% prettify nocode tag=pre+code %}
-$ pub outdated [options]
+$ dart pub outdated [options]
 {% endprettify %}
 
-Use `pub outdated` to identify out-of-date [package dependencies][]
+Use `dart pub outdated` to identify out-of-date [package dependencies][]
 and get advice on how to update them.
 [Best practices for dependency management][best practices]
 include using the most recent stable package versions,
 so you can get the latest bug fixes and improvements.
 
 {{site.alert.version-note}}
-  The `pub outdated` command was introduced in Dart 2.8.
+  The `dart pub outdated` command was introduced in Dart 2.10.
 {{site.alert.end}}
 
+{% comment %}
+Dart version updated acording with /tools/dart-tool/ page
+{% endcomment %}
+
 ## Overview
 
-Here's how you can use `pub outdated` to help you
+Here's how you can use `dart pub outdated` to help you
 update the dependencies of a package that you own
 (whether it's an app or library package):
 
 1. If your package doesn't have a `pubspec.lock` file
    checked into source control,
-   **run `pub get`** in the top directory of the package —
+   **run `dart pub get`** in the top directory of the package —
    the directory that contains your package's
   [`pubspec.yaml`](/tools/pub/pubspec) file.
-1. **Run `pub outdated`**
+1. **Run `dart pub outdated`**
    to identify which package dependencies are out-of-date.
    Note the affected packages,
    so that later you can test the behavior of code that uses them.
-1. Follow the recommendations of `pub outdated` for updating the packages.
-   Some updates might require only running `pub upgrade`.
-   Others might require updating `pubspec.yaml` before running `pub upgrade`.
-1. **Run `pub outdated`** to confirm that you're using
+1. Follow the recommendations of `dart pub outdated` for updating the packages.
+   Some updates might require only running ` dart pub upgrade`.
+   Others might require updating `pubspec.yaml` before running `dart pub upgrade`.
+1. **Run `dart pub outdated`** to confirm that you're using
    the latest compatible package versions.
 1. **Test** your package to confirm that it still works as expected.
 
 You might still have out-of-date dependencies due to
 [transitive dependencies][].
 If you want to determine the cause,
-try running [`pub deps`][] and searching the output for
+try running [`dart pub deps`][] and searching the output for
 the name of each out-of-date package.
 
 
 ## Example
 
-Here's an example of running `pub outdated` on
+Here's an example of running `dart pub outdated` on
 an example that has several out-of-date dependencies.
 Three of the dependencies (`args`, `http`, and `path`) are direct,
 and one is transitive (`meta`).
 As the following example shows,
-`pub outdated` colorizes the output by default
+`dart pub outdated` colorizes the output by default
 when you run it on the command line.
 
-{% include_relative _pub-outdated-output.html %}
+{% include_relative _dart-pub-outdated-output.html %}
```

Comment:
"_pub-outdated-ouput.html" references to a file on the relative path. So I think you should not change it to "_dart-pub-outdated-ouput.html" as there is no file with this name on the relative path